quic/s2n-quic-core/src/recovery/loss.rs

```diff
@@ -0,0 +1,173 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{packet::number::PacketNumber, time::Timestamp};
+use core::time::Duration;
+
+//= https://www.rfc-editor.org/rfc/rfc9002#section-6.1.1
+//# The RECOMMENDED initial value for the packet reordering threshold
+//# (kPacketThreshold) is 3, based on best practices for TCP loss
+//# detection [RFC5681] [RFC6675].  In order to remain similar to TCP,
+//# implementations SHOULD NOT use a packet threshold less than 3; see
+//# [RFC5681].
+pub const K_PACKET_THRESHOLD: u64 = 3;
```

Comment:
Would it be worth accepting this an a parameter, since this is just a default?

Reply:
Yeah I can do that, the time threshold is already a parameter so it makes sense